.gitattributes

```diff
@@ -0,0 +1,45 @@
+# Automatically normalize line endings.
+* text=auto
+
+# Files and directories to exclude from the generated archive (export-ignore).
+# These will be omitted from GitHub/Packagist ZIP archives to keep packages small.
+
+# Tests and test fixtures
+/tests/ export-ignore
+/tests/** export-ignore
+
+# PHP unit / static analysis configs.
+/phpcs.xml.dist export-ignore
+/phpstan.neon.dist export-ignore
+/phpunit.xml.dist export-ignore
+
+# Continuous Integration configs.
+/.github/ export-ignore
+/.github/** export-ignore
+
+# Node and build artifacts.
+node_modules/ export-ignore
+node_modules/** export-ignore
+package.json export-ignore
+package-lock.json export-ignore
+
+# Composer/vendor and dev tooling.
+/vendor/ export-ignore
+/vendor/** export-ignore
```

Comment:
I think the line above will cover for the second (aka I think we can remove this second line of `/vendor/** export-ignore`)

Reply:
I dont think we need to export-ignore anything that's already `.gitignore`d